src/database/index.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from '@firebase/database';
```

Comment:
This is pretty slick. Does typing generation work properly with this?

Reply:
From my tests yes. I gave an overview of how I came to this conclusion in the PR description:

> `npm test` passes, and `npm run integration` passes tests but hangs (not sure if that's an issue from my side or not). Moreover, performed sanity test using:
> `npm run build:dist && cp ./lib/firebase-admin-8.12.1.tgz ./ && ./.github/scripts/verify_package.sh
./firebase-admin-8.12.1.tgz`
> I also found tsc works fine as well in the temporary folder, and I loaded the folder in vscode and types seem to be working as expected.

I need to still address the hanging

Reply:
For visibility: I fixed the hanging last week, it was mostly due to not calling the delete function properly on the `DatabaseService`.